kge/model/embedder/lookup_embedder.py

```diff
@@ -25,10 +25,11 @@ def __init__(
 
         # read config
         self.normalize_p = self.get_option("normalize.p")
-        self.regularize = self.check_option("regularize", ["", "lp"])
+        self.regularize = self.check_option("regularize", ["", "lp", "n3"])
         self.sparse = self.get_option("sparse")
         self.config.check("train.trace_level", ["batch", "epoch"])
         self.vocab_size = vocab_size
+        self.vector_space = self.check_option("vector_space", ["real", "complex"])
```

Comment:
check that n3 is only enabled when space is marked "complex"

Reply:
Please let me know if my implementation is exactly what you are looking for or not ^^

If it's exactly what you are looking for, should I raise an error when a user uses N3 in euclidian space?

Reply:
Yes, but this should be the case already with your implementaiton.